test/cases/enumBoolean.ts

```diff
@@ -0,0 +1,23 @@
+export var schema = {
+  "title": "EnumBoolean",
+  "type": "object",
+  "properties": {
+      "foo": {
+        "type" : "boolean",
```

Comment:
Sure - that makes sense.  The code will break without it as it stands now. But I think it can be refactored to work.